src/genotyper.cc

```diff
@@ -272,21 +255,19 @@ static Status translate_genotypes(const genotyper_config& cfg, const unified_sit
 
     // reference allele maps if it contains the unified site
     range rng(record);
-    if (rng.contains(site.pos)) {
+    if (rng.overlaps(site.pos)) {
```

Comment:
Relaxed due to assumptions made on genotyping: `record` should be the only variant record in this site, so as long as it overlaps with `site.pos`, we can assume that any other region not covered by `rng` will be just REF.

[Note: currently, cases of multiple variant records overlapping a site still goes into this code path, which I'm planning to refactor out, but those cases should be rendered as loss due to `fill_allele` routine which safeguards against multiple records]